core/lib/core.ts

```diff
@@ -800,7 +800,8 @@ export function trackerCore(base64: boolean, callback?: (PayloadData) => void) {
 		 *
 		 * @todo make `nodeName` enum
 		 */
-		trackFormChange: function (
+		trackFormFocusOrChange: function (
```

Comment:
should we separate `trackFormChange` from `trackFormFocus`?

Reply:
I thought it would make sense to keep them together as they send exactly the same data. But feel free to edit this PR as you see fit - it was just a quick implementation to help us identify which forms needed more attention